projects/gloo/pkg/plugins/staged_filter_util.go

```diff
@@ -1,17 +1,23 @@
 package plugins
 
 import (
+	"errors"
+
 	envoyhttp "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
 	"github.com/golang/protobuf/proto"
 	"github.com/solo-io/gloo/projects/gloo/pkg/utils"
 )
 
-func NewStagedFilter(name string, stage FilterStage) StagedHttpFilter {
-	s, _ := NewStagedFilterWithConfig(name, nil, stage)
-	return s
+// NewStagedFilterWithConfig is deprecated as config is now always needed
```

Comment:
[nit] Can we mark this as deprecated following GoDoc conventions? 

```suggestion
// Deprecated: config is now always needed
```